lib/url-plugin.js

```diff
@@ -12,9 +12,11 @@
  */
 import { promises as fs } from 'fs';
 import { basename } from 'path';
+import { imageSize } from 'image-size';
 
 const defaultOpts = {
   prefix: 'url',
+  imagePrefix: 'img-url',
```

Comment:
Cheeky suggestion: Wouldn’t it keep this plugin simpler if instead of a second prefix it would take a list of file extensions that we consider images?

Reply:
Hmm, that would make the typing a lot harder. RIght now the module definition is based on prefixes. What you're proposing would depend on both a prefix and a (set of) suffixes, and I don't think TS supports that.

I'll keep it explicit for now, but if the typing can be done, happy for it to change.